workflows/Enrich_InsightIDR_Alerts_With_Threat_Intelligence_From_VirusTotal_To_Slack/workflow.spec.yaml

```
@@ -0,0 +1,48 @@
+extension: workflow
+products: ["insightconnect", "insightidr"]
+name: Enrich_InsightIDR_Alerts_With_Threat_Intelligence_From_VirusTotal_To_Slack
+title: "Enrich InsightIDR Alerts With Threat Intelligence From VirusTotal To Slack"
```

I believe if this is changed to 'with' and rebuilt it should fix the build failure
```suggestion
title: "Enrich InsightIDR Alerts with Threat Intelligence From VirusTotal To Slack"
```

---

@ablakley-r7 If I change to `with` then validator is throwing issue that `From` should be also lowercase. Changing to `from` makes validator to pass - this naming version was released and approved 13th April but as I mentioned above I was asked by @blaxminarayan-r7 here: https://rapid7.slack.com/archives/G012ZKN6PCZ/p1650902675506859?thread_ts=1650898853.155939&cid=G012ZKN6PCZ to change it so now validator is failing.

---

Sorry completely missed that comment, all looks great then!